app/views/posts/index.html.erb

```diff
@@ -1 +1,46 @@
-<h1>This is the posts file for testing</h1>
\ No newline at end of file
+<div style="margin: 20px 40px;">
+  <div class="card mb-3" style="border: none;">
+    <div class="row g-0" style="gap: 1.5%;">
+      <div class="col-md-4" style="width: 70px;">
+        <img src ='<%= @user.photo %>' class="img-fluid rounded" alt="User Photo">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body" style="border: 1px solid black; margin-top: 5px; width: 1345px;">
+          <h5 class="card-title"><%= @user.name %></h5>
+          <p class="card-text"><small class="text-muted">
+            Number of posts: <%= @user.posts_counter %></small>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <% @posts.each do |post|%>
+    <div class="card border-success mb-3">
+      <div class="card-body text-success">
+        <a href = <%= user_post_path(post.author.id, post.id) %> style="text-decoration: none;">
+          <h5 class="card-title">Post <%= @posts.index(post) + 1%></h5>
+        </a>
+        <p class="card-text">
+          <%= post.text %>
+        </p>
+        <div class= "card-text" style="float: right">
+          <span class="card-text">Comments: <%= post.comments_counter%></span>
+          <span class="card-text">Likes: <%= post.likes_counter%></span>
+        </div>
+      </div>
+      <% if post.comments.any? %>
```

Comment:
https://github.com/Zafron047/Blog_app_by_Rails/blob/4d771a9d4026028ac9027b89d2b3fc2798a5441e/app/models/post.rb#L22-L24
- [ ] Well done for showing some recent comments 👏, however, in the project requirements, you are expected to render a post's **5 most recent comments** on the posts index page but you are only rendering 4. I noticed that you tweaked your method to allow users to set the limit, however, please set the default `limit = 5` 👍 
![image](https://github.com/Zafron047/Blog_app_by_Rails/assets/46839548/03681a69-b36d-4d24-9cae-aa80cea12331)

Reply:
I have also removed the extra loop that was rendering all the comments after loading the 5 recent comments.

![5most_recent_comments](https://github.com/Zafron047/Blog_app_by_Rails/assets/117600135/f2966e8b-9ca0-48e1-a7b0-c42504040066)